### DISCUSSION TITLE
可以将【姿态及手】单出一个吗~这个插件的手部识别实在太好用了

### DISCUSSION THREAD

**Discussion by GonDesign** [huchenlei/sd-webui-openpose-editor#56] [Ideas]
![image](https://github.com/huchenlei/sd-webui-openpose-editor/assets/125015943/f90a585e-8935-4ccf-aabf-79d44cabd202)
![Dingtalk_20231130155908](https://github.com/huchenlei/sd-webui-openpose-editor/assets/125015943/0962af1e-4c68-4d3d-b135-0e7c0e1b96d5)
参考Openpse~非常需要将姿态及手单独控制，期待回复~~

**Comment by huchenlei** [comment]
Are you proposing adding `dw_pose_hand` option? (Body + Hands)

**Reply by GonDesign** [reply]
yes！
